### ISSUE TITLE
can't ready property of uploadform error on import

### ISSUE THREAD

**Issue by natsumi** [contentful/contentful-import#60]
I managed to succesfully export one of my spaces but on import I get this error:

```
ERR! import TypeError: Cannot read property 'uploadFrom' of null
ERR! import     at /Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/contentful-batch-libs/dist/transform/transformers.js:45:23
ERR! import     at /Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/lodash/_baseReduce.js:18:9
ERR! import     at /Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/lodash/_createBaseFor.js:17:11
ERR! import     at baseForOwn (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/lodash/_baseForOwn.js:13:20)
ERR! import     at /Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/lodash/_createBaseEach.js:17:14
ERR! import     at baseReduce (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/lodash/_baseReduce.js:15:3)
ERR! import     at reduce (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/lodash/reduce.js:48:10)
ERR! import     at Object.assets (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/contentful-batch-libs/dist/transform/transformers.js:43:57)
ERR! import     at /Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/contentful-batch-libs/dist/transform/transform-space.js:17:40
ERR! import     at tryCatcher (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/util.js:16:23)
ERR! import     at MappingPromiseArray._promiseFulfilled (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/map.js:61:38)
ERR! import     at MappingPromiseArray.PromiseArray._iterate (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/promise_array.js:114:31)
ERR! import     at MappingPromiseArray.init (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/promise_array.js:78:10)
ERR! import     at MappingPromiseArray._asyncInit (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/map.js:30:10)
ERR! import     at Async._drainQueue (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/async.js:138:12)
ERR! import     at Async._drainQueues (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/async.js:143:10)
ERR! import     at Immediate.Async.drainQueues (/Users/aseng/.nvm/versions/node/v6.11.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/async.js:17:14)
```

**Comment by axe312ger** [comment]
Hey Natsumi,

looks like you got unprocessed assets in your space. Check your exported json for `"uploadFrom": null` or sth similar. Navigate to that asset at https://app.contentful.com/ and make sure it is processed. Reuploading might also help.

All the best,
Benedikt

**Comment by axe312ger** [comment]
Could be fixed now, still some cases remain where this might happen. We are working on it.

**Comment by axe312ger** [comment]
@natsumi does it still happen to you?

**Comment by tmarciszewski** [comment]
I'm getting the same error. After exporting from space 6ms84vgrdq84 (i had a problem with exporting - was able to export without assets) and trying to import into newly created space er2cqihzfk30 I'm getting error mentioned above.

**Comment by axe312ger** [comment]
@tmarciszewski if you get the exact same error, your export.json does contain faulty assets. Try to delete them all or track down the faulty one via the asset id you will get from the errors log.

We are working on payload validation to make this more clear, see #81

**Comment by tmarciszewski** [comment]
Error log does not seem to be very helpful
```
[
  {
    "ts": "2018-02-09T10:15:26.508Z",
    "level": "error",
    "error": {
      "stacktrace": [
        "TypeError: Cannot read property 'uploadFrom' of null",
        "/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/contentful-batch-libs/dist/transform/transformers.js:45:24",
        "/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/lodash/_baseReduce.js:18:9",
        "/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/lodash/_createBaseFor.js:17:11",
        "baseForOwn (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/lodash/_baseForOwn.js:13:20)",
        "/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/lodash/_createBaseEach.js:17:14",
        "baseReduce (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/lodash/_baseReduce.js:15:3)",
        "reduce (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/lodash/reduce.js:48:10)",
        "Object.assets (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/contentful-batch-libs/dist/transform/transformers.js:43:57)",
        "/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/contentful-batch-libs/dist/transform/transform-space.js:18:40",
        "tryCatcher (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/util.js:16:23)",
        "MappingPromiseArray._promiseFulfilled (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/map.js:61:38)",
        "MappingPromiseArray.PromiseArray._iterate (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/promise_array.js:114:31)",
        "MappingPromiseArray.init (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/promise_array.js:78:10)",
        "MappingPromiseArray._asyncInit (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/map.js:30:10)",
        "Async._drainQueue (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/async.js:138:12)",
        "Async._drainQueues (/Users/tomasz.marciszewski/.nvm/versions/node/v9.2.0/lib/node_modules/contentful-import/node_modules/bluebird/js/release/async.js:143:10)"
      ]
    }
  }
]
```

**Comment by axe312ger** [comment]
We are about to merge https://github.com/contentful/contentful-import/pull/81 which will identify the faulty assets before the actual import is run.

I'll update you when we released a new version.

**Comment by phoebeschmidt** [comment]
This has been released since v6.2.0 on Feb 20, 2017 🎉
